cedar-example-use-cases/document_cloud/policies.cedarschema

```diff
@@ -1,60 +1,42 @@
-entity DocumentShare, Drive;
+entity Drive {
+  owner: User,
+};
 entity Document  = {
-  "isPrivate": Bool,
-  "manageACL": DocumentShare,
-  "modifyACL": DocumentShare,
+  "manageACL": Group,
+  "modifyACL": Group,
   "owner": User,
   "publicAccess": String,
-  "viewACL": DocumentShare,
+  "viewACL": Group,
 };
-entity Group in [DocumentShare] = {
+entity Group = {
   "owner": User,
 };
-entity Public in [DocumentShare];
-entity User in [Group] = {
-  "blocked": Set<User>,
+entity User in Group = {
+  blocked?: Set<User>,
   "personalGroup": Group,
 };
 
 action DeleteGroup, ModifyGroup appliesTo {
   principal: [User],
   resource: [Group],
-  context: {
-    "is_authenticated": Bool,
-  }
 };
 action CreateGroup appliesTo {
   principal: [User],
   resource: [Drive],
-  context: {
-    "is_authenticated": Bool,
-  }
 };
 action ViewDocument appliesTo {
-  principal: [User, Public],
+  principal: [User],
   resource: [Document],
-  context: {
-    "is_authenticated": Bool,
-  }
 };
-action AddToShareACL, DeleteDocument, EditIsPrivate, EditPublicAccess appliesTo {
+action AddToShareACL, DeleteDocument, EditPublicAccess appliesTo {
```

Comment:
Nit: in the Cedar docs, we usually use camelCase for action names

Reply:
TIL